apps/nova/src/pages/Plans.tsx

```diff
@@ -0,0 +1,45 @@
+import PageTitle from "@/components/Common/PageTitle";
+import Content from "@/components/Plans/Content";
+import { PlansDataProps } from "@/components/Plans/types";
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+import { useEffect, useState } from "react";
+
+export const Plans: React.FC = () => {
+  const intl = useFormatMessage();
+  const [plans, setPlans] = useState<PlansDataProps | null>(null);
```

Comment:
Becuase you will not need to fetch the data, you can assign a default value based on the current tab.

Reply:
Also, no need to have the plans state here. You can import it directly in the "Content" component.